docs/metrics/customizing-the-sdk/README.md

```diff
@@ -473,11 +572,11 @@ recording of `Exemplar`s.
 OpenTelemetry SDK comes with the following `ExemplarFilter`s (defined on
 `ExemplarFilterType`):
 
-* (Default behavior) `AlwaysOff`: Makes no measurements eligible for becoming an
+- (Default behavior) `AlwaysOff`: Makes no measurements eligible for becoming an
```

Comment:
please remove all unrelated changes from this PR. We are glad to accept them as a separate PR, so as to keep each PR focused on only one aspect only.